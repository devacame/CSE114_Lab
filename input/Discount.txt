100.00
5