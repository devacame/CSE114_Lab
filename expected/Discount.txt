Discount: 20%
Discounted price: $80.00 with discounted amount per unit: $20.00 from the original price of $100.00
Final total price: $80.00 * 5 = $400.00.